app/javascript/dashboard/routes/dashboard/conversation/ContactPanel.vue

```diff
@@ -90,10 +90,14 @@
         icon="ion-clock"
       />
     </div>
+    <a v-show="!currentChat.muted" class="contact--mute" @click="mute">
+      Mute Contact
```

Comment:
Please add this in i18n files, add a new key (CONTACT_PANEL.MUTE_CONTACT)

Reply:
How should I add this for languages other than English? Would adding `"MUTE_CONTACT": ""` suffice?

Reply:
You don't need to worry about other languages, just don't add the key in those files. The fallback `en` language will be taken in those cases.